### ISSUE TITLE
Use f-strings

### ISSUE THREAD

**Issue by jirassimok** [jirassimok/sky-python-music-sheet-maker#2]
It requires Python 3.6, but it would make a bunch of code look a lot nicer.

**Comment by jmmelko** [comment]
> It requires Python 3.6, but it would make a bunch of code look a lot nicer.

Do you mean all the parts where we do ugly things like this?
` "string" + str(number) + "string"`

**Comment by jirassimok** [comment]
Yeah. For example, that particular line could be replaced by `f"string{number}string"`.

You can do a lot of other formatting, like aligning numbers and using `repr`, and you can get the same effects a few different ways using `str.format`. The documentations on [format string syntax](https://docs.python.org/3/library/string.html#format-string-syntax) is the primary source, if you like very-detailed explanations.
